### DISCUSSION TITLE
Is there a Vuetify's v-slide-group in Quasar??

### DISCUSSION THREAD

**Discussion by Sjoerd82** [quasarframework/quasar#15460] [General - Components / Directives / etc] [answered]
I'm attempting to migrate from Vuetify to Quasar. In general, I find that Quasar has a _superset_ of Vuetify's components. However, my app heavily depends on Vuetify's **V-Slide-Group**. It uses this mostly in mobile mode, so it needs to support touch too. Like so:

![image](https://user-images.githubusercontent.com/15013786/219852928-75dc0af7-2dc3-43c2-8654-58d2efa5009c.png)

So far I've not seen this component in Quasar's list of components, maybe I'm overlooking it?
Or is there an alternative for this? (a quick search reveals that there's a thing called swiper.js)

This thing worries me, as this is a potentially big obstacle for migrating from Vuetify. I hope my worries are unfounded :-)

**Comment by metalsadman** [comment, marked as answer]
try horizontal scroll area https://quasar.dev/vue-components/scroll-area#example--horizontal-content
then add whatever content in it, you can also add the the chevrons your self, using this example https://quasar.dev/vue-components/scroll-area#example--scroll-position.

rough example combining what I stated above https://codepen.io/metalsadman/pen/BaOozZZ?editors=1010

**Reply by Sjoerd82** [reply]
Perfection! 🤘Thanks!

**Reply by Sjoerd82** [reply]
Except for one small detail :-)

When scrolling, the scrollbar briefly appears (Chrome). I see that `scrollbar-width: none;` is set, but in Chrome inspector it's crossed out, and seems to not be applied. Maybe due to WCAG considerations? I didn't have much time to experiment with it now though, will try some more stuff later.

**Reply by metalsadman** [reply]
If you want it to always show just use visible prop in your qscrollarea.

**Reply by Sjoerd82** [reply]
Nah, I'd prefer to not see the scrollbar there at all.

**Reply by metalsadman** [reply]
You can play with the bar-style, make it more opaque, change it's color (transparent), make it reactive to react to certain conditions  if you wish and etc...
